halo2_proofs/src/plonk/circuit.rs

```diff
@@ -1169,17 +1169,18 @@ impl<F: Field> ConstraintSystem<F> {
         panic!("get_instance_query_index called for non-existent query");
     }
 
-    pub(crate) fn get_any_query_index(&self, column: Column<Any>, at: Rotation) -> usize {
+    pub(crate) fn get_any_query_index(&self, column: Column<Any>) -> usize {
```

Comment:
Is this really only used to query at `Rotation::cur()`? If so, we should document it as such.